### PR TITLE
SORDEV-7868

### PR DESCRIPTION
**Pull request by pk-sgent**
Fixes #

### CODE REVIEW

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java [hunk 1]
```diff
@@ -80,6 +81,209 @@ public EditCaseSteps(
...
+        () ->
+            webDriverHelpers.clickWebElementByText(
+                OUTCOME_OF_CASE_OPTIONS, CaseOutcome.getValueFor("UNKNOWN").toUpperCase()));
+
+    When(
+        "I check if date of outcome filed is available",
+        () -> webDriverHelpers.clickOnWebElementBySelector(DATE_OF_OUTCOME));
+
+    When(
+        "I click on yes option in Sequelae",
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java:131]: I click on yes option in Sequelae  all these methods can be parametrized in BDD with Yes/No/Unknown as strings

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java [hunk 2]
```diff
@@ -80,6 +81,209 @@ public EditCaseSteps(
...
+                SEQUELAE_OPTIONS, CaseOutcome.getValueFor("No").toUpperCase()));
+    When(
+        "I click on unknown option in Sequelae",
+        () ->
+            webDriverHelpers.clickWebElementByText(
+                SEQUELAE_OPTIONS, CaseOutcome.getValueFor("Unknown").toUpperCase()));
+
+    When(
+        "I check if Sequelae Details field is available",
+        () -> webDriverHelpers.clickOnWebElementBySelector(SEQUELAE_DETAILS));
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java:148]: the method says check but the action is click. Can we refine it a bit?

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java [hunk 3]
```diff
@@ -80,6 +81,209 @@ public EditCaseSteps(
...
+        () -> webDriverHelpers.clickOnWebElementBySelector(COMMUNITY_COMBOBOX_BY_PLACE_OF_STAY));
+
+    When(
+        "I click on Facility as place of stay",
+        () ->
+            webDriverHelpers.clickWebElementByText(
+                PLACE_OF_STAY_OPTIONS, CaseOutcome.getValueFor("Facility").toUpperCase()));
+
+    When(
+        "I check if Facility Category combobox is available",
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java:179]: same check and click

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java [hunk 4]
```diff
@@ -80,6 +81,209 @@ public EditCaseSteps(
...
+        () -> webDriverHelpers.clickOnWebElementBySelector(FACILITY_TYPE_COMBOBOX));
+
+    When(
+        "I set Facility as a Other Facility",
+        () ->
+            webDriverHelpers.selectFromCombobox(
+                FACILITY_HEALTH_COMBOBOX, CaseOutcome.getValueFor("Other facility")));
+
+    When(
+        "I check if Facility name and description field is available",
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java:193]: same

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java (hunk 2 above)
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java]: waitUntilElementIsVisibleAndClickable

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java [hunk 5]
```diff
@@ -80,6 +81,209 @@ public EditCaseSteps(
...
+
+    When(
+        "I check if district combobox is available and i select Responsible District",
+        () -> {
+          aCase = caseService.buildEditGeneratedCase();
+          webDriverHelpers.selectFromCombobox(PLACE_OF_STAY_DISTRICT_COMBOBOX, aCase.getDistrict());
+        });
+
+    When(
+        "I check if community combobox is available",
```
[the remark this answers is not in the document]
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java:134]: waitUntilElementIsVisibleAndClickable

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java (hunk 3 above)
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java:146]: waitUntilElementIsVisibleAndClickable

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java (hunk 4 above)
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java:160]: waitUntilElementIsVisibleAndClickable

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java [hunk 6]
```diff
@@ -80,6 +81,209 @@ public EditCaseSteps(
...
+        () ->
+            webDriverHelpers.selectFromCombobox(
+                FACILITY_HEALTH_COMBOBOX, CaseOutcome.getValueFor("Other facility")));
+
+    When(
+        "I check if Facility name and description field is available",
+        () -> webDriverHelpers.clickOnWebElementBySelector(PLACE_DESCRIPTION_INPUT));
+
+    When(
+        "I set Quarantine Home",
```
[the remark this answers is not in the document]
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java]: I set Quarantine ([^"]*)

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**